lib/config/options/index.ts

```diff
@@ -2416,21 +2416,23 @@ const options: RenovateOptions[] = [
   },
   {
     name: 'concurrentRequestLimit',
-    description: 'Limit concurrent requests per host.',
+    description:
+      'Limit concurrent requests per host. Set to `-1` for no limit. Default value is 10 concurrent requests.',
```

Comment:
```suggestion
      'Limit concurrent requests per host. Set to `-1` for no limit.',
```

What does 0 designate?